purchase_request/tests/test_purchase_request.py

```diff
@@ -117,6 +117,8 @@ def test_purchase_request_status(self):
         # Unlink purchase_lines from state approved
         with self.assertRaises(UserError):
             purchase_request_line.unlink()
+        purchase = purchase_request_line.purchase_lines.write({"price_unit": 100.0})
+        self.assertEqual(purchase, True)
```

Comment:
And the rest for the same.

```suggestion
        purchase_request_line.purchase_lines.price_unit = 100.0
```

Reply:
Thanks, I've already adjusted it.